examples/batch_statistical.ipynb

```diff
@@ -0,0 +1,1030 @@
+  {
+   "cell_type": "code",
+   "execution_count": 39,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "rgb_evalscript = \"\"\"\n",
+    "//VERSION=3\n",
+    "\n",
+    "function setup() {\n",
+    "  return {\n",
+    "    input: [\n",
+    "      {\n",
+    "        bands: [\n",
+    "          \"B04\",\n",
+    "          \"B08\",\n",
+    "          \"dataMask\"\n",
+    "        ]\n",
+    "      }\n",
+    "    ],\n",
+    "    output: [\n",
+    "      {\n",
+    "        id: \"ndvi\",\n",
+    "        bands: 1\n",
+    "      },\n",
+    "      {\n",
+    "        id: \"dataMask\",\n",
+    "        bands: 1\n",
+    "      }\n",
+    "    ]\n",
+    "  }\n",
+    "}\n",
+    "\n",
+    "function evaluatePixel(samples) {\n",
+    "    return {\n",
+    "      ndvi: [index(samples.B08, samples.B04)],\n",
+    "      dataMask: [samples.dataMask]\n",
+    "    };\n",
+    "}\n",
+    "\"\"\"\n",
+    "\n",
+    "aggregation = SentinelHubStatistical.aggregation(\n",
+    "    evalscript=rgb_evalscript,\n",
+    "    time_interval=(\"2020-06-01\", \"2020-06-30\"),\n",
+    "    aggregation_interval=\"P1D\",\n",
+    "    resolution=(10, 10),\n",
+    ")\n",
+    "\n",
+    "histogram_calculations = {\"ndvi\": {\"histograms\": {\"default\": {\"nBins\": 20, \"lowEdge\": -1.0, \"highEdge\": 1.0}}}}\n",
+    "\n",
+    "input_data = [SentinelHubStatistical.input_data(DataCollection.SENTINEL2_L1C, maxcc=0.8)]"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Now we define the storage specifications. For this we can use helpe method `SentinelHubBatchStatistical.s3_specification` that generates a dictionary specifying an S3 access. One can also write all of the dictionaries by hand, as long as they follow the [official specifications](https://docs.sentinel-hub.com/api/latest/reference/#operation/createNewBatchStatisticsRequest)."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 43,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "input_features = SentinelHubBatchStatistical.s3_specification(INPUT_PATH, AWS_ID, AWS_SECRET)\n",
+    "output = SentinelHubBatchStatistical.s3_specification(OUTPUT_PATH, AWS_ID, AWS_SECRET)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### 2.2 Creating the request\n",
+    "\n",
+    "We first create a Batch Statistical client with which we'll manage any requests."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 44,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "client = SentinelHubBatchStatistical()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "We then create a new request with the `create` method. If we have any `SentinelHubStatistical` requests on hand, we can use them to create an *upgraded* batch request via the `create_from_request` method. The geometry of the original request is then substituted with the input file."
```

Comment:
"upgraded batch request" sounds weird. I would just say that we can instead of `create` method use a helper method `create_from_request`.